spec.bs

```diff
@@ -655,7 +760,8 @@ To <dfn>append private state token redemption request headers</dfn> given a [=/r
             the issuer's most recent commitments.
  1. Let |token| be the result of [=retrieve a token|retrieving a token=] for |issuer|.
  1. If |token| is null, return.
- 1. Let |record| be the result of running a cryptographic redemption procedure with |token|. If the procedure fails, return.
+ 1. Let |redeemRequest| be an empty [=RedeemRequest=].
```

Comment:
`RedeemRequest` is a struct. Does step mean with empty members?

Reply:
Yes.